packages/typespec-test/test/openai_generic/generated/typespec-ts/src/api/audio/transcriptions/index.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  CreateTranscriptionRequest,
+  CreateTranscriptionResponse,
+} from "../../../models/models.js";
+import {
+  AudioTranscriptionsCreate200Response,
+  AudioTranscriptionsCreateDefaultResponse,
+  isUnexpected,
+  OpenAIContext as Client,
+} from "../../../rest/index.js";
+import {
+  StreamableMethod,
+  operationOptionsToRequestParameters,
+} from "@azure-rest/core-client";
+import { AudioTranscriptionsCreateOptions } from "../../../models/options.js";
+
+export function _audioTranscriptionsCreateSend(
+  context: Client,
+  audio: CreateTranscriptionRequest,
+  options: AudioTranscriptionsCreateOptions = { requestOptions: {} }
+): StreamableMethod<
+  | AudioTranscriptionsCreate200Response
+  | AudioTranscriptionsCreateDefaultResponse
+> {
+  return context
+    .path("/audio/transcriptions")
+    .post({
+      ...operationOptionsToRequestParameters(options),
+      contentType: (options.contentType as any) ?? "multipart/form-data",
+      body: {
+        file: audio["file"],
+        model: audio["model"],
+        prompt: audio["prompt"],
+        response_format: audio["responseFormat"],
+        temperature: audio["temperature"],
+        language: audio["language"],
+      },
+    });
+}
+
+export async function _audioTranscriptionsCreateDeserialize(
+  result:
+    | AudioTranscriptionsCreate200Response
+    | AudioTranscriptionsCreateDefaultResponse
+): Promise<CreateTranscriptionResponse> {
+  if (isUnexpected(result)) {
+    throw result.body;
+  }
+
+  return {
+    text: result.body["text"],
+  };
+}
+
+export async function audioTranscriptionsCreate(
```

Comment:
This is incorrectly renamed, we shouldn't rename the api layer operations if hierarchy clients is enable which should by default be enabled.